.github/workflows/tests.yml

```diff
@@ -415,13 +416,13 @@ jobs:
         # The 2010 image is the most recent spec that comes with Python 2.7,
         # and only up through the tag 2021-02-06-3d322a5
         env:
-          DOCKER_IMAGE: quay.io/pypa/${{ matrix.image }}:2021-02-06-3d322a5
+          DOCKER_IMAGE: quay.io/pypa/${{ matrix.image }}
```

Comment:
This change appears to be dropping support for building Python 2.7 wheels. Is that deliberate?

Reply:
That old pinned version does not contain a Python 3.10. The pin was removed in https://github.com/zopefoundation/meta/commit/3e329394c636b6ef844221acb7f3ba1a0b496e3c. I do not think it is worth the effort to have multiple docker file versions to get wheels for outdated Python versions. (Python 3.5 are also no longer built.)